recipes/VoxCeleb/SpeakerRec/hparams/train_ecapa_tdnn.yaml

```diff
@@ -80,67 +86,88 @@ epoch_counter: !new:speechbrain.utils.epoch_loop.EpochCounter
     limit: !ref <number_of_epochs>
 
 
-augment_wavedrop: !new:speechbrain.lobes.augment.TimeDomainSpecAugment
-    sample_rate: !ref <sample_rate>
-    speeds: [100]
-
-augment_speed: !new:speechbrain.lobes.augment.TimeDomainSpecAugment
-    sample_rate: !ref <sample_rate>
-    speeds: [95, 100, 105]
```

Comment:
same ?

Reply:
This was done on purpose. I think applying speed change to speaker recognition is not a  good idea as it will change the pitch and the speaker identity as well.